.github/workflows/reusable-industrial-ci-with-cache.yml

```
@@ -15,6 +15,11 @@ on:
         description: 'UPSTREAM_WORKSPACE variable for industrial_ci. Usually path to local .repos file.'
         required: true
```

```suggestion
        required: false
```
otherwise we can't use it with [this workflow configuration](https://github.com/ros-controls/control.ros.org/blob/0799e0d85de792e6659cf33d72fe4dfd36c936ad/.github/workflows/rolling-binary-build-testing.yml#L13-L20)? Or maybe adding `upstream_workspace: ''` would be sufficient?

---

Making it optional does seem more reasonable to me.